detectors/node/opentelemetry-resource-detector-azure/src/detectors/AzureFunctionsDetector.ts

```diff
@@ -49,28 +50,23 @@ const AZURE_FUNCTIONS_ATTRIBUTE_ENV_VARS = {
 class AzureFunctionsDetector implements DetectorSync {
   detect(): IResource {
     let attributes = {};
-    const functionName = process.env[WEBSITE_SITE_NAME];
+    const serviceName = process.env[WEBSITE_SITE_NAME];
     const functionVersion = process.env[FUNCTIONS_VERSION];
-    if (functionName && functionVersion) {
+    if (serviceName && functionVersion) {
       const functionInstance = process.env[WEBSITE_INSTANCE_ID];
       const functionMemLimit = process.env[FUNCTIONS_MEM_LIMIT];
 
       attributes = {
         [SEMRESATTRS_CLOUD_PROVIDER]: CLOUDPROVIDERVALUES_AZURE,
         [SEMRESATTRS_CLOUD_PLATFORM]: CLOUDPLATFORMVALUES_AZURE_FUNCTIONS,
         [SEMRESATTRS_CLOUD_REGION]: process.env[REGION_NAME],
+        [SEMRESATTRS_PROCESS_PID]: process.pid,
       };
 
-      if (functionName) {
+      if (serviceName) {
         attributes = {
           ...attributes,
-          [SEMRESATTRS_FAAS_NAME]: functionName,
-        };
-      }
-      if (functionVersion) {
-        attributes = {
-          ...attributes,
-          [SEMRESATTRS_FAAS_VERSION]: functionVersion,
```

Comment:
Can you share the motivation to remove this attribute?

Reply:
The `FUNCTIONS_EXTENSION_VERSION` will be set twice, once by the worker process and again by the host process in Azure Functions. This is a problem because the worker process will have a different (and not the value we want to set for the `FAAS_VERSION` value). Until a solution is determined for this, we shouldn't record this attribute.